File: test/data/src/java/CumulFROM.java

cumulTTFromZC_65 = 0.0D;
cumulTTFromZC_39 = 0.0D;
cumulTTFromZC_91 = 0.0D;
if (calendarMoments.contains("Anthesis")){
    if (switchMaize == 0)
        cumulTTFromZC_65 = cumulTT-calendarCumuls.get(calendarMoments.indexOf("Anthesis"));
}    
if (calendarMoments.contains("FlagLeafLiguleJustVisible")){
    if (switchMaize == 0)
        cumulTTFromZC_39 = cumulTT-calendarCumuls.get(calendarMoments.indexOf("FlagLeafLiguleJustVisible"));
}   
if (calendarMoments.contains("EndGrainFilling")){
    if (switchMaize == 0)
        cumulTTFromZC_91 = cumulTT-calendarCumuls.get(calendarMoments.indexOf("FlagLeafLiguleJustVisible"));
}

File: test/data/src/cs/CumulFROM.cs

cumulTTFromZC_65 = 0.0D;
cumulTTFromZC_39 = 0.0D;
cumulTTFromZC_91 = 0.0D;  
if (calendarMoments.Contains("Anthesis")){
    if (switchMaize == 0)
        cumulTTFromZC_65 = cumulTT-calendarCumuls[calendarMoments.IndexOf("Anthesis")];
}    
if (calendarMoments.Contains("FlagLeafLiguleJustVisible")){
    if (switchMaize == 0)
        cumulTTFromZC_39 = cumulTT-calendarCumuls[calendarMoments.IndexOf("FlagLeafLiguleJustVisible")];
}   
if (calendarMoments.Contains("EndGrainFilling")){
    if (switchMaize == 0)
        cumulTTFromZC_91 = cumulTT-calendarCumuls[calendarMoments.IndexOf("FlagLeafLiguleJustVisible")];
}